numpy/core/_add_newdocs.py

```diff
@@ -3953,13 +3953,23 @@
 
     Examples
     --------
-    For a 1D array, ``a.tolist()`` is almost the same as ``list(a)``:
+    For a 1D array, ``a.tolist()`` is almost the same as ``list(a)``, except that it changes numpy scalars to Python scalars:
```

Comment:
line length should be limited to 80 characters. Also, maybe better as "except that ``tolist`` changes ...